FreeRTOS/FreeRTOS-Kernel
# Update version number in manifest.yml

Description
-----------
This PR adds automation in auto_release.yml for updating version number in Manifest.yml file

Test Steps
-----------
Run Kernel auto release from forked repository successfully, with updated version number in Manifest.yml file.

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
